### PR TITLE
Test for did:dht signed VCs + release

### PR DESCRIPTION
**Pull request by phoebe-lew**

### CODE REVIEW

File: packages/credentials/tests/verifiable-credential.spec.ts
Comment (decentralgabe) [inline packages/credentials/tests/verifiable-credential.spec.ts:284]:
```suggestion
      const privateKey = (await Jose.jwkToKey({ key: signingKeyPair.privateKeyJwk! })).keyMaterial;
```

**Review by decentralgabe — APPROVED** [review]
approving pending small comments

**Comment by frankhinek** [comment]
> approving pending small comments

Approving even though all of the tests are failing? 🙃

File: packages/agent/package.json
Comment (frankhinek) [inline packages/agent/package.json:72]: `@web5/agent` still needs to be updated before using `@web5/crypto` version `0.2.3` so reverting.
Details here: https://github.com/TBD54566975/web5-js/pull/318
```suggestion
    "@web5/crypto": "0.2.2",
```

File: packages/api/package.json
Comment (frankhinek) [inline packages/api/package.json:80]: `@web5/api` needs to be updated before using `@web5/crypto` version `0.2.3` so reverting.
Details here: https://github.com/TBD54566975/web5-js/pull/318
```suggestion
    "@web5/crypto": "0.2.2",
```

File: packages/credentials/package.json
Comment (frankhinek) [inline packages/credentials/package.json:88]: `@web5/credentials` needs to be updated before using `@web5/crypto` version `0.2.3` so reverting.
Details here: https://github.com/TBD54566975/web5-js/pull/318
```suggestion
    "@web5/crypto": "0.2.2",
```

File: packages/dids/package.json
Comment (frankhinek) [inline packages/dids/package.json:80]: `@web5/dids` needs to be updated before using `@web5/crypto` version `0.2.3` so reverting.
Details here: https://github.com/TBD54566975/web5-js/pull/318
```suggestion
    "@web5/crypto": "0.2.2",
```

**Comment by decentralgabe** [comment]
@frankhinek if our pre commit hooks work properly then merging is blocked until they pass. the contents of the PR seem correct even if there are minor fixable test issues.

File: packages/proxy-agent/package.json
Comment (frankhinek) [inline packages/proxy-agent/package.json:72]: `@web5/proxy-agent` needs to be updated before using `@web5/crypto` version `0.2.3` so reverting.
Details here: https://github.com/TBD54566975/web5-js/pull/318
```suggestion
    "@web5/crypto": "0.2.2",
```

**Review by frankhinek — CHANGES_REQUESTED** [review]
Since we're going to release `@web5/api` version `0.8.3` later this week per Angie's announcement [on Discord](https://discord.com/channels/937858703112155166/984873257666768936/1178826478549487807) the `version` string in `packages/api/package.json` should also be set to `0.8.2` for this PR.

**Comment by decentralgabe** [comment]
@frankhinek I understand your perspective, but respectfully disagree. This is the value of having multiple reviewers with different areas of expertise. Our review system worked as desired, as we noticed different issues with the changes, and the changes were not able to be merged without them being addressed (via hooks and your change request).

I have high trust in my colleagues to address requested changes and merge when they feel sufficient review has been completed (and CI passes, as is enforced).

File: packages/api/package.json
Comment (KendallWeihe) [inline packages/api/package.json:3]:
```suggestion
  "version": "0.8.4",
```

right?

Comment (frankhinek) [reply packages/api/package.json:3]: no bump down to `0.8.2` to avoid triggering a premature release of that package until Dev Programs completes their testing and the dev community has had a couple days notice that the hosted DWN servers will be bounced.

Comment (KendallWeihe) [reply packages/api/package.json:3]: Oh my b, I see you commented on that earlier, I jumped right into code diff view

**Review by frankhinek — APPROVED** [review]

**Pull request merged**
